content/en/blog/2022/cni-mode/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Merbridge CNI Mode"
+linkTitle: "Merbridge CNI Mode"
+date: 2022-05-18
+weight: 1
+description: This blog explains how CNI works in Merbridge.
+---
+
+<!--Merbridge CNI 模式的出现，旨在能够更好地适配服务网格的功能。之前没有 CNI 模式时，Merbridge 能够做得事情比较有限。其中最大的问题是不能适配注入 Istio 的 Sidecar Annotation，这就导致 Merbridge 无法排除某些端口或 IP 段的流量等。同时，由于之前 Merbridge 只处理 Pod 内部的连接请求，这就导致，如果是外部发送到 Pod 的流量，Merbridge 将无法处理。-->
+The CNI mode is designed to make Merbridge work better with service mesh features. Before having this mode, Merbridge cannot exclude traffic from certain ports or IP ranges, because Istio Annotations configured for a pod cannot take effect in Merbridge. Besides, it can only process connection requests in pods. Traffic from external to pods cannot be processed.
```

Comment:
```suggestion
The CNI mode is designed to better adapt to the service mesh functions. Before having this mode, Merbridge was limited to certain scenarios. The biggest problem was that it could not adapt to the sidecar annotations from the container injected by Istio, which led to Merbridge cannot exclude traffic from certain ports and IP ranges. Furthermore, Merbridge was only able to handle requests inside the pod,  which means the external traffic sent to the pod was not handled.
```